components/accelerator/nvidia/persistence-mode/component_output.go

```diff
@@ -58,6 +61,8 @@ const (
 	StateKeyPersistenceModeData       = "data"
 	StateKeyPersistenceModeEncoding   = "encoding"
 	StateValueMemoryUsageEncodingJSON = "json"
+
+	initializeGracePeriod = 3 * time.Minute
```

Comment:
Should we define constant values ​​somewhere to avoid duplication? Also, 3 minutes is a bit long. Do we have past data that 3 minutes is reasonable? 1 minute might be enough.

Reply:
our control plane collects every 1 min and I checked some historical data seems it will still fail on second collection, 3 min is about 3 collections, just to be secure..

Reply:
Got it